### PR TITLE
Updated UCSCGI and BDC Logos - DOCK-2204/2291

### PR DESCRIPTION
**Pull request by KimberleyChong**
**Description**
Updated UCSCGI and BDC Logos.  Since we did not have permission to use the BDC logos I created a [new](https://www.figma.com/file/5OVaqhEGBcbJHVR0ZLZ9pT/BDC-Text-Logo?node-id=0%3A1&t=C7LaPaMm6mpAjxLF-1) text logo to better follow their guidelines (linked on the ticket). I also needed to update the `diagram-launch-with` image on the landing page since all of the logos are embedded in the actual photo.

**Launch-with - Before:**
image
**Launch-with - After:**
image
**Special Thanks - Before:**
image
**Special Thanks - After:**
image

**Footer - Before:**
image
**Footer - After:**
image

I added a `max-width` restriction in the images because the new UCSCGI logo kept overflowing to create a 3rd row which I didn't think was necessary.

**Important to Note:**
The logos used by the organizations on Dockstore would need to be updated by the Organizer/Owner _themselves_ as they are not part of the UI, not sure if those would need to be addressed or not, but I would think we are not responsible for updating them?
image
image

**Review Instructions**
Ensure the logos on the Landing page, About page and Footer look good :)

**Issue**
[DOCK-2204
](https://ucsc-cgl.atlassian.net/browse/DOCK-2204)[GitHub](https://github.com/dockstore/dockstore/issues/5019)
[DOCK-2291](https://ucsc-cgl.atlassian.net/browse/DOCK-2291)
[GitHub](https://github.com/dockstore/dockstore/issues/5262)

**Security**
If there are any concerns that require extra attention from the security team, highlight them here.

Please make sure that you've checked the following before submitting your pull request. Thanks!

- [x] Check that your code compiles by running `npm run build`
- [x] Ensure that the PR targets the correct branch. Check the milestone or fix version of the ticket.
- [x] If this is the first time you're submitting a PR or even if you just need a refresher, consider reviewing our [style guide](https://github.com/dockstore/dockstore/wiki/Dockstore-Frontend-Opinionated-Style-Guide#pr-checklist)
- [x] Do not bypass Angular sanitization (bypassSecurityTrustHtml, etc.), or justify why you need to do so
- [x] If displaying markdown, use the `markdown-wrapper` component, which does extra sanitization
- [x] Do not use cookies, although this may change in the future
- [x] Run `npm audit` and ensure you are not introducing new vulnerabilities
- [x] Do due diligence on new 3rd party libraries, checking for CVEs
- [x] Don't allow user-uploaded images to be served from the Dockstore domain
- [x] If this PR is for a user-facing feature, create and link a documentation ticket for this feature (usually in the same milestone as the linked issue). Style points if you create a documentation PR directly and link that instead.
- [x] Check whether this PR disables tests. If it legitimately needs to disable a test, create a new ticket to re-enable it in a specific milestone. 

[DOCK-2291]: https://ucsc-cgl.atlassian.net/browse/DOCK-2291?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by denis-yuen** [comment]
> The logos used by the organizations on Dockstore would need to be updated by the Organizer/Owner _themselves_ as they are not part of the UI, not sure if those would need to be addressed or not, but I would think we are not responsible for updating them?

This is correct

**Review by denis-yuen — APPROVED** [review]
(should probably get more than one PR approval from UCSC developers)

**Review by kathy-t — APPROVED** [review]
Do we need to update the NHLBI BioData Catalyst launch with button? 

(Screenshot from this [workflow](https://dockstore.org/workflows/github.com/DataBiosphere/topmed-workflows/UM_variant_caller_wdl:1.32.0?tab=info))
image

**Comment by denis-yuen** [comment]
> Do we need to update the NHLBI BioData Catalyst launch with button?

FYI @bvizzier

**Comment by KimberleyChong** [comment]
> Do we need to update the NHLBI BioData Catalyst launch with button?
> 
> (Screenshot from this [workflow](https://dockstore.org/workflows/github.com/DataBiosphere/topmed-workflows/UM_variant_caller_wdl:1.32.0?tab=info)) image

Mmm thank you, I'll add the Registered Trademark

**Comment by KimberleyChong** [comment]
**Launch with button update:**
image

File: src/app/workflow/launch-third-party/launch-third-party.component.ts [hunk 1]
```diff
@@ -166,7 +166,7 @@ export class LaunchThirdPartyComponent extends Base implements OnChanges, OnInit
 
   bdCatalystSevenBridgesTooltip$: Observable<string> = combineLatest([this.hasContent$, this.hasHttpImports$]).pipe(
     map(([hasContent, hasHttpImports]) =>
-      this.sevenBridgesTooltip(hasContent, hasHttpImports, 'NHLBI BioData Catalyst powered by Seven Bridges')
+      this.sevenBridgesTooltip(hasContent, hasHttpImports, 'BioData Catalyst (BDC) Powered by Seven Bridges')
```
Comment (KimberleyChong) [inline src/app/workflow/launch-third-party/launch-third-party.component.ts:169]: Also updated some language:
image
To match:
image

Comment (coverbeck) [reply src/app/workflow/launch-third-party/launch-third-party.component.ts:169]: Super nit: I prefer a lower case "p" in `powered`; but if it's in the guidelines...

**Review by coverbeck — APPROVED** [review]

**Comment by bvizzier** [comment]
> > The logos used by the organizations on Dockstore would need to be updated by the Organizer/Owner _themselves_ as they are not part of the UI, not sure if those would need to be addressed or not, but I would think we are not responsible for updating them?
> 
> This is correct

Please file a ticket and assign it to me for the UCSC Genomics Ticket Org.

**Review by bvizzier — APPROVED** [review]
Looks good to me.  Thanks!

**Review by svonworl — APPROVED** [review]

**Review by svonworl — APPROVED** [review]

**Pull request merged**